src/components/exchange/ExchangeAssetList.tsx

```diff
@@ -339,18 +351,18 @@ const ExchangeAssetList = (
           renderSectionHeader={ExchangeAssetSectionListHeader}
           scrollsToTop={isFocused}
           sections={itemsWithFavorite}
+          keyExtractor={keyExtractor}
+          scrollIndicatorInsets={scrollIndicatorInsets}
+          windowSize={7}
+          style={{ height: '100%' }}
```

Comment:
i tried removing these 2 and didn't have any issue, i get they're coming from https://github.com/rainbow-me/rainbow/pull/4145/files#diff-bd05a3fa6c538d0edc86d14263910d93c227be6cfff87245e57547e3b1709c1dL112 but WDYT?

Reply:
Yeah I just tested on my end and there was no rendering difference, Ive updated to remove